tests/test_psi.py

```diff
@@ -0,0 +1,10 @@
+"""
+Test code in src/psi
+"""
+
+from src.psi.util import compute_psi
+
+def test_compute_psi_returns_correct_indices():
```

Comment:
Can we use pytest mark_parametrize and include a few edge cases, such as having no intersection

Reply:
That's a good idea! I'll add some more tests